html-templates/sugcon-site/stylesheets/components/column-container - Copy.css

```diff
@@ -0,0 +1,22 @@
+/* line 1, ../../sass/components/column-container - Copy.scss */
+.column-container {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+/* line 5, ../../sass/components/column-container - Copy.scss */
+.column-container h2 {
+  padding-top: 50px;
+  padding-bottom: 10px;
+}
+
+/* line 11, ../../sass/components/column-container - Copy.scss */
+.hero.medium + .column-container {
+  padding-top: 70px;
+}
+
+@media (max-width: 767.98px) {
+  /* line 15, ../../sass/components/column-container - Copy.scss */
+  .hero.medium + .column-container {
+    padding-top: 40px;
+  }
+}
```

Comment:
Seems like a duplication of column-contianer.css?

Reply:
Ah sorry, I should look before answering;  above statements still stands though and I think we should refactor styling at some point (to a setup similar to my old helix example imo) 

I'll remove the - Copy.css .. not sure where that came from :)

Reply:
There are really 2 css files with" - Copy" in their name (cta and column-container) both are identical to other files and are both in the html templates and under wwwroot.

These files are in the current main too so this is unrelated to the SUGCON sites. They are really just "inherited" when the styling was copied - see https://github.com/Sitecore/MVP-Site/blob/bc1b34d5904f2dfc6e9797a5d2660a1c43842520/src/Project/MvpSite/rendering/wwwroot/stylesheets/components/column-container%20-%20Copy.css

I've deleted them and pushed change to this PR

Reply:
Thanks for tidying this, agree to could do with having a general tidy up on things like this across the MVP site as well